README.md

```diff
@@ -1,4 +1,60 @@
-# Week 5 (October 11th 2018)
+# Week 5
 
-## Reset Your Project
-`git checkout -f week4`
+## House Keeping
+
+1. Thinking about your Projects!
+2. Make sure you know how to get to your Projects when you create them on
+   your computer.
+
+## Lets get setup first!!
+
+`git checkout -f week4`.
+
+This will blow away all your changes, so if there is something you want to keep for later go ahead and commit it and remove the `-f` from your command.
+
+## What are Components?
+
+First lets talk about what components are in terms of “theory” before we dig into how React Implements a component.
+
+## Lets Get Started!
+
+This is a summary of what we will be doing, but its _best to be in class_ to get the detailed play by play. You can see the end result in week5!
+
+1. Create Two Components: `People` and `Person`.
+2. Work from the ground up with components, start with the smallest building blocks. Its alright if you work from the top to bottom as well, up to you. Can you guess which component is the smallest building block?
+3. Output an `` in the Person component that display a `name` passed as a prop.
```

Comment:
Not sure what is missing, but something's missing

Reply:
Output a div maybe?